tests/unit/checkmate/models/db/mixins_test.py

```diff
@@ -30,6 +30,13 @@ def test_it_with_limit(self, db_session):
         assert items.count() == 1
         assert items[0].hash in ["hash_1", "hash_3"]
 
+    @pytest.fixture(autouse=True, scope="class")
+    def create_test_table_with_hash(self, db_engine):
+        self.TableWithHash.__table__.drop(db_engine, checkfirst=True)
+        self.TableWithHash.__table__.create(db_engine)
+        yield
+        self.TableWithHash.__table__.drop(db_engine)
```

Comment:
Same as in LMS: there are ORM classes defined in some of the tests and so these now need a fixture to create them.

We don't need the test grouping thing in Checkmate because it doesn't run its tests in parallel. Checkmate's tests are few and fast enough that running them in parallel would be slower, not faster. We actually _can't_ add the test group decorator because `pytest-xdist` isn't installed.

Unfortunately if anyone one day does want to change Checkmate's tests to run in parallel they're going to run into this issue and will have to figure out that the tests in this class need to be grouped.